sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -683,6 +685,8 @@ def receive_deferred_messages(
                 :caption: Receive deferred messages from ServiceBus.
 
         """
+        if kwargs:
+            warnings.warn(f"Unsupported keyword args: {kwargs}")
```

Comment:
Is the `f {}` formatting support in 3.6+?

Reply:
yes it is